### PR TITLE
Added support for Romanian translation

### PR DESCRIPTION
**Pull request by Achime11**

### CODE REVIEW

File: Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"Nitrox_JoinServerIpAddress": "Adresa IP a serverului",
+	"Nitrox_JoinServerPassword": "Parola:",
+	"Nitrox_JoinServerFailed": "Nu te poți alătura serverului",
+	"Nitrox_Cancel": "Anulare",
+	"Nitrox_Join": "Alătură-te",
+	"Nitrox_PlayerColor": "Culoarea jucătorului",
+	"Nitrox_EnterName": "Introdu numele de jucător",
+	"Nitrox_SubmitPassword": "Trimite parola",
+	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat ...",
+
```
Comment (Coding-Hen) [inline Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:19]: Minor: remove spaces between as to keep consistency with other language files

**Review by Coding-Hen — APPROVED** [review]

File: Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json (hunk 1 above)
Comment (Achime11) [reply Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:19]: Fixed

File: Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	
+	// In game menu
+	"Nitrox_LoadingScreenWarn": "Așteptă-te la bug-uri care pot strica jocul",
+	"Nitrox_WorldSettling": "În așteptarea stabilirii lumii",
+	"Nitrox_JoiningSession": "Alăturarea la sesiune Multiplayer",
+	"Nitrox_SyncingWorld": "Sincronizarea Lumii Multiplayer",
+	"Nitrox_LostConnection": "Conexiune pierdută la serverul jocului",
+	"Nitrox_PlayerKicked": "Ai fost dat afară de pe server",
+		
+	// In game logging
```
Comment (Coding-Hen) [inline Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:28]:
```suggestion
	"Nitrox_PlayerKicked": "Ai fost dat afară de pe server",
	// In game logging
```

File: Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	"Nitrox_JoinServerPassword": "Parola:",
+	"Nitrox_JoinServerFailed": "Nu te poți alătura serverului",
+	"Nitrox_Cancel": "Anulare",
+	"Nitrox_Join": "Alătură-te",
+	"Nitrox_PlayerColor": "Culoarea jucătorului",
+	"Nitrox_EnterName": "Introdu numele de jucător",
+	"Nitrox_SubmitPassword": "Trimite parola",
+	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat...",
+	
+	// In game menu
```
Comment (Coding-Hen) [inline Nitrox.Subnautica.Assets/LanguageFiles/Romanian.json:20]:
```suggestion
	"Nitrox_MultiplayerLoaded": "Clientul Multiplayer încărcat...",
	// In game menu
```

**Comment by dartasen** [comment]
Hi, thanks for the translation.
We updated the translation system in #1364 and we added new translations, is it possible to update this translation with the new entries ?

See [New english file](https://github.com/dartasen/Nitrox/blob/translations/Nitrox.Subnautica.Assets/LanguageFiles/English.json)
Those were added : 
```json
"Nitrox_JoinServerPasswordHeader": "Server password required",
"Nitrox_DisconnectedSession": "Disconnected from server",
"Nitrox_FirewallInterfering": "Seems like your firewall settings are interfering",
```

**Review by Coding-Hen — CHANGES_REQUESTED** [review]
Sorry I didnt completly explain. Lets remove these spaces to be consistent with #1364

**Comment by Achime11** [comment]
> Hi, thanks for the translation.
> We updated the translation system in #1364 and we added new translations, is it possible to update this translation with the new entries ?
> 
> See [New english file](https://github.com/dartasen/Nitrox/blob/translations/Nitrox.Subnautica.Assets/LanguageFiles/English.json)
> Those were added :
> 
> ```json
> "Nitrox_JoinServerPasswordHeader": "Server password required",
> "Nitrox_DisconnectedSession": "Disconnected from server",
> "Nitrox_FirewallInterfering": "Seems like your firewall settings are interfering",
> ```

**Review by Coding-Hen — CHANGES_REQUESTED** [review]
Lets unindent this to keep consistant formatting with #1364

**Pull request merged**